OutOfSchool/OutOfSchool.IdentityServer/Controllers/AccountController.cs

```diff
@@ -196,6 +196,33 @@ public async Task<IActionResult> SendConfirmEmail()
         return Ok();
     }
 
+    [HttpGet]
+    public async Task<IActionResult> ReSendEmailConfirmation()
+    {
+        var user = await userManager.FindByEmailAsync(User.Identity.Name);
+
+        var token = await userManager.GenerateEmailConfirmationTokenAsync(user);
+        var returnUrl = "Login";
+        var callBackUrl = Url.Action("EmailConfirmation", "Account", new { token, user.Email, returnUrl }, Request.Scheme);
+
+        var email = user.Email;
+        var subject = localizer["Confirm email"];
+        var userActionViewModel = new UserActionViewModel
+        {
+            FirstName = user.FirstName,
+            LastName = user.LastName,
+            ActionUrl = callBackUrl,
+        };
+        var content = await renderer.GetHtmlPlainStringAsync(RazorTemplates.ConfirmEmail, userActionViewModel);
+        await emailSender.SendAsync(email, subject, content);
+
+        var path = $"{HttpContext.Request.Path.Value}[{HttpContext.Request.Method}]";
```

Comment:
`var path = $"{HttpContext.Request.Path.Value}[{HttpContext.Request.Method}]";` is used pretty everywhere. Isn't it a right time to add at least private method for it (or even controller extension method if usages span across this controller?)